akita/src/queryEntity.ts

```diff
@@ -301,9 +301,9 @@ export class QueryEntity<S extends EntityState, EntityType = getEntityType<S>, I
    *
    *  this.query.selectEntityAction();
    */
-  selectEntityAction(action: EntityActions): Observable<ID[]>;
-  selectEntityAction(): Observable<EntityAction<EntityType>>;
-  selectEntityAction(action?: EntityActions): Observable<ID[] | EntityAction<EntityType>> {
+  selectEntityAction(action: EntityActions): Observable<IDType[]>;
+  selectEntityAction(): Observable<EntityAction<IDType>>;
```

Comment:
The second one should be EntityType

Reply:
What do you mean? `selectEntityAction()` never returns `EntityType` right?